src/sagemaker/local/image.py

```diff
@@ -98,6 +110,68 @@ def __init__(self, instance_type, instance_count, image, sagemaker_session=None)
         self.container_root = None
         self.container = None
 
+    def process(
+        self, processing_inputs, processing_output_config, environment, processing_job_name
+    ):
+        """Run a processing job locally using docker-compose.
+
+        Args:
+            processing_inputs (dict): The processing input specification.
+            processing_output_config: The processing output configuration specification.
```

Comment:
indicate type of this, like the other args?

Reply:
Done.